mifospay/src/main/java/org/mifos/mobilewallet/mifospay/bank/fragment/DebitCardFragment.java

```diff
@@ -44,6 +46,8 @@
     @BindView(R.id.pe_year)
     PinEntryEditText mPeYear;
 
+    boolean key = false;
```

Comment:
Working seems fine but @EGOR-IND can you please explain the use of this field in your code and maybe try to give a more meaningful name to this field

Reply:
I used this field because when the length of mPeYear becomes zero then we don't want the cursor to go directly on the month field so when it's true then cursor will go to the month field

Reply:
Hey @devansh-299 don't merge it i found a small flaw. Let me make some improvements.

Reply:
Hey @devansh-299, now this PR is ready without any flaw and the use of field that you asked for have same use as it has before.